lib/src/instances/option.dart

```diff
@@ -84,11 +228,103 @@ class Some<T> extends Option<T> {
   }
 
   @override
+  bool contains(Object element) => element == value;
+
+  @override
+  T elementAt(int index) {
+    if (index != 0) {
+      throw new RangeError.index(index, this, "index", null, 1);
+    }
+
+    return value;
+  }
+
+  @override
+  bool every(bool f(T element)) => f(value);
+
+  @override
+  IterableMonad<S> expand<S>(Iterable<S> f(T element)) =>
```

Comment:
Well, `f` returns an `Iterable` in an `Iterable`, but its signature and its behavior is pretty much as `flatMap`. I'm not completely convinced but I like the idea that `expand` to be an alias of `flatMap` although it also makes sense to have it as an `Iterable` bridge from Options. I'm not sure which one is better.